codersdk/insights.go

```diff
@@ -0,0 +1,189 @@
+package codersdk
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+	"golang.org/x/xerrors"
+)
+
+const InsightsTimeLayout = time.RFC3339
```

Comment:
👍

Reply:
Are you in favor of keeping this? I believe @mtojek was of the opposite mind. 😄

Reply:
or at least move it to coderd and unexpose

Reply:
It can't be moved to `coderd` because it's also used in this package, we can keep it unexposed and duplicated in both, or removed entirely.

Reply:
We could do that, I was more in favour of specifying that particular layout and no other.